### PR TITLE
Improvements to the sponsored transaction tutorial

### PR DESCRIPTION
**Pull request by DOBEN**
## Purpose

- Fix broken link.
- Add screenshots/explanation to differentiate between `normal transaction vs. sponsored transaction`

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

**Review by abizjak — APPROVED** [review]
Good graphics.

File: source/mainnet/smart-contracts/tutorials/sponsoredTransactions/index.rst [hunk 1]
```diff
@@ -41,6 +41,24 @@ offering to submit transactions on behalf of the user on-chain.The third-party
...
+
+The user creates a normal transaction by signing its ``account`` address, its ``nonce``, and the ``action``. The ``nonce`` increases sequentially every time the user sends a transaction to the blockchain to prevent replay attacks. The nonce is of type u64 (8 bytes) and by design >= 1.
+
+.. image:: ./images/NormalSmartContractTransactionFlow.png
+   :alt: Normal Smart Contract Transaction Flow
+   :align: center
+
+A sponsored transaction decouples the transaction fee authorization (third-party wallet) from the action authorization (user's wallet).
+The user signs a message (in that case the prepend is ``account`` address and 8 zero bytes) to authorize the ``action``. Hence, the 8 zero bytes ensure that the user does not accidentally sign a valid transaction. This means the red signature is not a transaction, while the black signature is a valid transaction in the below screenshot. The user's signature is sent to the backend server.
```
Comment (dg-concordium) [inline source/mainnet/smart-contracts/tutorials/sponsoredTransactions/index.rst:56]:
```suggestion
The user signs a message (in that case the prepend is ``account`` address and 8 zero bytes) to authorize the ``action``. Hence, the 8 zero bytes ensure that the user does not accidentally sign a valid transaction. This means the red signature is not a transaction, while the black signature is a valid transaction in the image below. The user's signature is sent to the backend server.
```

Comment (dg-concordium) [inline source/mainnet/smart-contracts/tutorials/sponsoredTransactions/index.rst:56]:
```suggestion
The user signs a message (in that case the prepend is ``account`` address and 8 zero bytes) to authorize the ``action``. Hence, the 8 zero bytes ensure that the user does not accidentally sign a valid transaction. This means the red signature is not a transaction, while the black signature is a valid transaction in the below screenshot. The user's signature is sent to the back-end server.
```

File: source/mainnet/smart-contracts/tutorials/sponsoredTransactions/index.rst [hunk 2]
```diff
@@ -41,6 +41,24 @@ offering to submit transactions on behalf of the user on-chain.The third-party
...
+The third-party creates a valid transaction at the backend server which is sent to the blockchain.
```
Comment (dg-concordium) [inline source/mainnet/smart-contracts/tutorials/sponsoredTransactions/index.rst:57]:
```suggestion
The third-party creates a valid transaction at the back-end server, which is sent to the blockchain.
```

**Review by dg-concordium — APPROVED** [review]
Just some small changes.

**Pull request merged**